### DISCUSSION TITLE
Change Web port from 443 to 4438

### DISCUSSION THREAD

**Discussion by Artapel** [snipe/snipe-it#14603] [General]
Morning all

Can someone direct me to instructions or a guide on how to change the web port from 443 to 4438.

Kind Regards
Corné

**Comment by ksgrg** [comment]
Hi @Artapel 
This can be done on your web server configuration. What operating system are you using?
You can simply change the listening port for the site or virtual host to 4438 in the configuration.

Kind Regards
